lib/klaviyo/apis/templates.rb

```diff
@@ -0,0 +1,105 @@
+module Klaviyo
+  class Templates < Client
+    TEMPLATES = 'email-templates'
+    CLONE = 'clone'
+    RENDER = 'render'
+    SEND = 'send'
+
+    # Retrieve all email templates available.
+    # @param person_id [String] the id of the profile
+    # @return [JSON] A dictionary with a data property that contains an array of all the
+    #   email templates. Each entry is a separate email template object. If no email templates exist,
+    #   the resulting array will be empty. This request should never return an error.
+    def self.get_templates
+      v2_request(HTTP_GET, TEMPLATES)
```

Comment:
The Templates API is a V1 API, so I believe you should use the `v1_request` - have you tested these methods locally?

Reply:
Oh how embarrassing. I spent a good deal of time Thursday trying to figure out why my integration wasn't working in my app, turns out I just hadn't tested thoroughly. Yikes. I'll fix that...

Reply:
I've pushed the change, but I'll let you know once I've fully tested it.